### ISSUE TITLE
save localy on server instead of download. 

### ISSUE THREAD

**Issue by vortex91** [deepjyoti30/ytmdl-web-v2#74]
# Feature Request

## Checklist

- [x] I've searched the bugtracker for similar feature requests including closed ones.
- [x] I will add the enhancement label to the issue.

## Description

I use this to save localy to my music folder.  Would be nice to just have it bulk save a playlist to my local folder (maybe in settings or config pointing to a pth)

**Comment by deepjyoti30** [comment]
@vortex91 I don't quite understand what you're asking for here? Are you asking for a feature to be able to download playlists at once through the web utility? If so, what is the role of a `server` as you have mentioned it on the title.

What I understand from the description is that you use `ytmdl-web` to download to your machine's music folder but you want to be able to download playlists to your music folder?

**Comment by rzalawad** [comment]
@deepjyoti30 Does ytmdl-web support downloading to a local folder based on some cmd line or environ param?

I'm looking for something similar where on a self hosted server, I can download a music file to either a preconfigured playlist, to the default playlist (all music) or download on the client that made the request.

Out of the items mentioned above, what would you say falls within the scope of this project? I am familiar with programming (but not JS programming unfortunately.) I can try to take a stab at it :)

**Comment by deepjyoti30** [comment]
@rzalawad I think `ytmdl` CLI is a perfect match for you. I am not sure if you are aware of it but `ytmdl web` is built on top of the CLI app.

It also has playlists support and a lot more features than the web version.

Check it here: https://github.com/deepjyoti30/ytmdl

**Comment by Snapyou2** [comment]
I agree that it would be awesome to have this function for people that are self-hosting their music. The ytmdl CLI works but sometimes I just want to have a clean GUI to access my server (especially from my phone) and grab some music easily.

Many thanks for all the work by the way, this is an really excellent job.
